readme.md

```diff
@@ -1459,6 +1459,27 @@ Type: `object`
 
 Handy information about a key that was pressed.
 
+###### key.name
```

Comment:
The TS docs and readme should be in sync.

Reply:
Pull request updated with changes to sync readme and TS docs